packages/odyssey-react-mui/src/ToastStack.tsx

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React from "react";
+import { memo, forwardRef, ForwardedRef, ReactElement } from "react";
+import { Box, Snackbar, ToastProps } from ".";
+
+export type ToastStackProps = {
+  children: ReactElement | Array<ReactElement>;
+};
+
+const ToastStack = forwardRef(
+  ({ children }: ToastStackProps, ref: ForwardedRef<HTMLDivElement>) => {
+    const modifiedChildren = React.Children.map(
+      children,
+      (child: ReactElement<ToastProps>) => {
+        return React.cloneElement(child, { isStatic: true });
+      }
+    );
+
+    return (
+      <Snackbar open={true}>
+        <Box ref={ref} display="flex" flexDirection="column-reverse" gap={2}>
```

Comment:
**Non-Blocking**

@KevinGhadyani-Okta, here's another place to move MUI props into the theme